src/main/java/com/ibm/devops/connect/CRPipeline/UploadBuild.java

```diff
@@ -262,5 +331,25 @@ public String getDisplayName() {
         public boolean isApplicable(Class<? extends AbstractProject> jobType) {
             return true;
         }
+
+        public ListBoxModel doFillInstanceBaseUrlItems(@QueryParameter String currentInstanceBaseUrl) {
+            // Create ListBoxModel from all projects for this AWS Device Farm account.
+            List<ListBoxModel.Option> baseUrls = new ArrayList<ListBoxModel.Option>();
+            List<Entry> entries = Jenkins.getInstance().getDescriptorByType(DevOpsGlobalConfiguration.class).getEntries();
+            String all = "Upload Build to All UCV Instances";
+            baseUrls.add(new ListBoxModel.Option(all, all, all.equals(currentInstanceBaseUrl)));
+            for (Entry entry : entries) {
+                // We don't ignore case because these *should* be unique.
+                baseUrls.add(new ListBoxModel.Option(entry.getBaseUrl(), entry.getBaseUrl(), entry.getBaseUrl().equals(currentInstanceBaseUrl)));
+            }
+            return new ListBoxModel(baseUrls);
+        }
+    }
+
+    private String removeTrailingSlash(String url) {
+        if (url.endsWith("/")) {
+            url = url.substring(0, url.length() - 1);
+        }
+        return url;
```

Comment:
This method was in another file as well.  This should be in a utility file and only defined once instead of defined in every file.